spec/services/search/classified_listing_spec.rb

```diff
@@ -0,0 +1,88 @@
+require "rails_helper"
+
+RSpec.describe Search::ClassifiedListing, type: :service, elasticsearch: true do
+  describe "::index" do
+    it "indexes a classified_listing to elasticsearch" do
+      classified_listing = FactoryBot.create(:classified_listing)
+      expect { described_class.find_document(classified_listing.id) }.to raise_error(Elasticsearch::Transport::Transport::Errors::NotFound)
+      described_class.index(classified_listing.id, id: classified_listing.id)
+      expect(described_class.find_document(classified_listing.id)).not_to be_nil
+    end
+  end
+
+  describe "::find_document" do
+    it "fetches a document for a given ID from elasticsearch" do
+      classified_listing = FactoryBot.create(:classified_listing)
+      described_class.index(classified_listing.id, id: classified_listing.id)
```

Comment:
This will change to

```ruby
described_class.index(classified_listing.id, classified_listing.serialized_search_hash) 
```
In a separate PR (Part 2 - [Index ClassifiedListing into Elasticsearch](https://github.com/thepracticaldev/dev.to/projects/6#card-32289256)). See @mstruve’s [comment](https://github.com/thepracticaldev/dev.to/pull/5919/files#r375849067) on #5919 for more details 💯